src/react/features/connector/tools/useOpenFolder.js

```diff
@@ -0,0 +1,31 @@
+import { useEffect } from 'react'
+import { useDispatch, useStore } from '../../../store'
+
+import { enterRoom } from '../../navigation/actions'
+import { setInactive } from '../../tools/actions'
+import { addPath } from '../actions'
+
+import { useTools } from '../../tools/hooks'
+
+import { connect } from '../client'
+import { channels } from '../../../../shared/constants'
+
+import * as buttons from './buttons'
```

Comment:
I would suggest that `import *` would be prohibited and we should always explicitly define what we are importing and put them to their own variables.

In this case I suggest:
```
import { GENERATE_DIOGRAPH } from './buttons'
```

Reply:
Or even:
```
import { roomButtons } from './buttons'
```

And then use `roomButtons.id` instead of `buttons.GENERATE_DIOGRAPH`.

Reply:
Yes, that fine. Also actionTypes should be changed.

Reply:
import { GENERATE_DIOGRAPH_BUTTON } from './buttons'

Reply:
#23

Reply:
Good catch with the actionTypes. Didn't notice them earlier but I prefer the same things for them as well: explicit names instead of `*`.

Part of #23 sounds good! 👍